sagemaker_processing/index.rst

```diff
@@ -12,4 +12,7 @@ Processing
    fairness_and_explainability/fairness_and_explainability
    fairness_and_explainability/fairness_and_explainability_jsonlines_format
    fairness_and_explainability/fairness_and_explainability_byoc
-   fairness_and_explainability/fairness_and_explainability_spark
\ No newline at end of file
+   fairness_and_explainability/fairness_and_explainability_spark
```

Comment:
Checking that Clarify Processing a type of SageMaker Processing job, rather than a separate feature using the same wording?

Reply:
Another name for Sagemaker Clarify is also fairness and explainability as it detect the bias during pre and post training which tell us how fair is the dataset and the model and it also check for explainability which tell us why model made such decision.